cpp-py-data-aggregation/db/aggregation_consumer.py

```diff
@@ -22,25 +21,27 @@ def get_iterations() -> int:
 def run_aggregation_consumer(timing_file: t.TextIO, list_length: int) -> None:
     comm = MPI.COMM_WORLD
     rank = comm.Get_rank()
+    logger = get_logger(f"Data Aggregation PY Consumer MPI Rank {rank}")
+    logger.debug(f"Initialized Rank")
 
     if rank == 0:
         logger.info("Connecting clients")
         print("Connecting clients")
 
     # Connect a client and save connection time
     constructor_start = MPI.Wtime()
-    client = Client(cluster=True, logger)
+    client = Client(cluster=True)
```

Comment:
This change means that stuff inside the SmartRedis library won't be tagged with the context of who called it. I'd really prefer you revert it